packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -49,5 +52,5 @@ TODO: Remove this section if there are not any general updates.
 
 ## Full commit history
 
-More details about changes and fixes are available from the
-[DevTools git log](https://github.com/flutter/devtools/commits/master).
+To find a complete list of changes in this release, check out the
+[DevTools git log](https://github.com/flutter/devtools/tree/v2.28.0).
```

Comment:
@CoderDake can we have our helper script that preps the next round of release notes also enter in the version number here when generating the content for NEXT_RELEASE_NOTES.md?

Reply:
yeah that should be easy enough
Issue created: https://github.com/flutter/devtools/issues/6366

Reply:
Is the `<number>` I put in the template not sufficient here? It seems the update version script replaces all instances of that with the new version number.

Reply:
Ah yes sorry I didn't notice that! This should be good to go. I'll close the issue since this should allow it to bump that version as part of the current process.